schema.sql

```diff
@@ -8,4 +8,7 @@ CREATE TABLE animals (
     neutered boolean,
     weigth_kg numeric,
     PRIMARY KEY (id)
-)
\ No newline at end of file
+);
+
+ALTER TABLE animals
+ADD species TEXT;
```

Comment:
- **[OPTIONAL]** Although TEXT is for strings I suggest using VARCHAR, to define the limit of characters 👇🏽 

`ALTER TABLE animals ADD species varchar(255);`